<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmGerenciarEmprestimos</class>
 <widget class="QDialog" name="FmGerenciarEmprestimos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gerenciamento de Empréstimos</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_emprestimos">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>631</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>382</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_add_emprestimos">
        <property name="text">
         <string>Adicionar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_excluir_emprestimos">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_editar_emprestimos">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_atualizar_emprestimos">
        <property name="text">
         <string>Atualizar tabela</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>631</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="txt_pequisar">
        <property name="text">
         <string>Pesquisar livros:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pesquisar_emprestimo"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_pesquisar_emprestimo">
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_emprestimo">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Todos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ativo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Concluído</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vencido</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
